frontend/src/lib/components/entityDetails/InfoItem.svelte

```diff
@@ -24,10 +25,25 @@ Used to show a label-content pair in a Candidate's basic information.
 ```
 -->
 
-<div class="infoItem grid grid-flow-col grid-cols-[min-content_auto] justify-start gap-md">
+<div class="grid justify-start gap-md {vertical ? 'vertical-grid' : 'horizontal-grid'}">
   <!-- pt-[0.3rem] matches the baselines of the small label and the content text -->
-  <div class="small-label w-min min-w-[7rem] pt-[0.3rem] text-left align-top">{label}</div>
+  <div
+    class="test-label small-label pt-[0.3rem] text-left align-top {vertical
+      ? 'w-max'
+      : 'min-w-[10rem]'}">
```

Comment:
I noticed that on the "big data refactoring" branch this is already set to 10rem! 👍